code/_onclick/adjacent.dm

```diff
@@ -123,7 +123,7 @@
 
 // This is necessary for storage items not on your person.
 /obj/item/Adjacent(atom/neighbor)
-	if(neighbor == loc || neighbor == loc.loc) //Item is in the neighbor or something that it holds.
+	if(neighbor == loc || neighbor == loc?.loc) //Item is in the neighbor or something that it holds.
```

Comment:
Which item without `loc` happened to get compared?

Reply:
*shrug